### PR TITLE
Rebased new migrations on the original migrations

### PR DESCRIPTION
**Pull request by Gavinok**
This resolves https://github.com/bcgov/DITP-DevOps/issues/138 by correcting the alembic migration error.

Before granular configuration was implemented the original migration file initial_db_tables_d925cb39480e.py was used.

Due to an over site this file was removed and a new complete_regeneration_bc503ce16d6d.py was produced.

While this would work for new deployments this broke backwards compatibility with existing deployments. For this reason we restored initial_db_tables_d925cb39480e.py and based updated_to_support_granular__e6afa1dce289.py on it to preserve this compatibility.

### CODE REVIEW

File: endorser/api/db/migrations/versions/updated_to_support_granular__e6afa1dce289.py
Comment (WadeBarnes) [inline endorser/api/db/migrations/versions/updated_to_support_granular__e6afa1dce289.py:88]: Since this new field is `NOT NULL` it will require a default value.  Otherwise you run into this; https://github.com/bcgov/DITP-DevOps/issues/138#issuecomment-1874093933

**Review by WadeBarnes — CHANGES_REQUESTED** [review]

File: endorser/api/db/migrations/versions/updated_to_support_granular__e6afa1dce289.py
Comment (WadeBarnes) [reply endorser/api/db/migrations/versions/updated_to_support_granular__e6afa1dce289.py:88]: https://docs.sqlalchemy.org/en/20/core/defaults.html

Comment (Gavinok) [reply endorser/api/db/migrations/versions/updated_to_support_granular__e6afa1dce289.py]: I have corrected this in the code and regenerated the migration

**Comment by WadeBarnes** [comment]
The updates allowed the migration to run successfully; https://github.com/bcgov/DITP-DevOps/issues/138#issuecomment-1875388008.  Waiting on confirmation that everything is working as expected before approving and merging the PR, in case there are additional adjustments that need to be made on the models.

**Comment by esune** [comment]
The issue was with the data in `ledger_txn_request` allowing `null` values to be stored in the database, and the deserialization step not accounting for that possibility. I pushed a couple of code tweaks that fix the issue - new image was deployed and tested (see https://github.com/bcgov/DITP-DevOps/issues/138#issuecomment-1881700122)

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**
